code/api/src/modules/user/update_user.test.js

```diff
@@ -0,0 +1,45 @@
+import request from 'supertest';
+import express from 'express';
+import graphqlHTTP from 'express-graphql';
+import schema from '../../setup/schema';
+import db from '../../setup/database';
+import models from '../../setup/models'
+
+
+describe('user mutations', () => {
+  let server;
+  beforeAll(() => {
+    server = express();
+
+    server.use(
+      '/',
+      graphqlHTTP({
+        schema: schema,
+        graphiql: false
+      })
+    )
+  })
+
+  it('can update user style', async () => {
+    const user =  await models.User.create({
+      name: "User",
+      email: "user@email.com",
+      password: 12345,
+      styleResult: null
+    })
+    const response = await request(server)
+      .post('/')
+      .send({
+        query: `mutation {userUpdate(id: ${user.id}, styleResult: \"Neat\") {id styleResult}}`
```

Comment:
Above is the specific query to update a user, in this case with the styleResult "Neat"